src/components/index.js

```diff
@@ -38,6 +42,7 @@ export {
   ThumbnailEntry,
   ToolbarSection,
   Tooltip,
+  AboutPreferencesModal,
```

Comment:
Why does it have the word Preferences in it? It's just an About page. There are no preferences.

Reply:
Done